### PR TITLE
Spdlog changes

### PR DESCRIPTION
**Pull request by sfujimoto22**
Changed printf statements to spdlog for Shooter, Vision, Spinner, and Drivetrain. Changed include statements for spdlog directory to use brackets instead of quotes.

### CODE REVIEW

File: 2021/Robot2021/src/main/cpp/subsystems/Shooter.cpp [hunk 1]
```diff
@@ -238,7 +241,7 @@ void Shooter::SetShooterSpeed(int level)
             m_targetOutput = frc::SmartDashboard::GetNumber("SH_SmartDashOutput", 0.0);
             break;
         default:
-            std::printf("2135: SH invalid velocity requested - %d\n", level);
+            spdlog::warn("2135: SH invalid velocity requested - {}", level);
```
Comment (ngupta-22) [inline 2021/Robot2021/src/main/cpp/subsystems/Shooter.cpp:244]: Remove the 2135 at the front of the statement

**Review by ngupta-22 — CHANGES_REQUESTED** [review]
Everything else looks good, just make sure to remove the 2135

File: 2021/Robot2021/src/main/cpp/subsystems/Shooter.cpp (hunk 1 above)
Comment (sfujimoto22) [reply 2021/Robot2021/src/main/cpp/subsystems/Shooter.cpp]: Thank you, done!

**Review by ngupta-22 — APPROVED** [review]
Perfect!

**Comment by sfujimoto22** [comment]
> Everything else looks good, just make sure to remove the 2135

Thank you

**Pull request merged**